src/dma/mod.rs

```diff
@@ -1059,6 +1059,11 @@ where
         }
     }
 
+    //### ge experiment
+    pub fn peripheral(&self) -> &PERIPHERAL {
+        &self.peripheral
+    }
+
```

Comment:
What's the point of this? Also the comment is a bit weird. 😅

Reply:
As I understand this is method on `Transfer` for accessing to peripheral payload.
But I can't say is this safe.